### ISSUE TITLE
Blank thumbnails are created from some ETDs

### ISSUE THREAD

**Issue by alawvt** [VTUL/vtechworks#492]
[filter-media](http://mirror.swem.wm.edu/DSDOC/wiki.duraspace.org/display/DSDOC5x/Mediafilters%2Bfor%2BTransforming%2BDSpace%2BContent.html) generates thumbnails using ImageMagick. Most thumbnails are generated correctly for ETDs. For instances, all ETDs loaded yesterday are fine. However some (~2%?) are created as blank images. Examples are 

- https://vtechworks.lib.vt.edu/handle/10919/74049
- https://vtechworks.lib.vt.edu/handle/10919/74884
- https://vtechworks.lib.vt.edu/handle/10919/74885

In each case, the pdf seems normal, with the title page as first page.

I deleted these three .jpgs and regenerating each thumbnails with a version of
```
sudo -u vtechworks /dspace/bin/dspace filter-media -i 10919/74049 -v
```
The output was
```
PROCESSING: bitstream 277797 (item: 10919/74049)
File: Painter_MS_D_2017.pdf.jpg
IM Image Param: /tmp/impdfthumb476910666847671755.pdf[0] -flatten /tmp/impdfthumb476910666847671755.pdf.jpg 
IM Thumbnail Param: /tmp/impdfthumb476910666847671755.pdf.jpg -thumbnail 80x80 /tmp/impdfthumb476910666847671755.pdf.jpg.jpg 
FILTERED: bitstream 277797 (item: 10919/74049) and created 'Painter_MS_D_2017.pdf.jpg'
```
The resulting files are still blank.

The document properties for these three original PDFs show
```
PDF Producer: Microsoft: Print to PDF
```
The Object Inspector in Acrobat Pro shows for the text on the first pages of these files:
```
Font: CIDFont+F1 Size 15.97 Type: OpenType TT
```
In comparison a pdf which resulted in a normal thumbnail has
```
Font: Arial MT Size 12.00 Type: OpenType TT
```
These CID fonts are composed. I don't know if that might have some bearing. It is interesting that these were all produced by Microsoft Print to PDF.

[ImageMagickThumbnailFilter.java](https://github.com/VTUL/vtechworks/blob/c720bb063a70bd7023530d9d434359fdebcda613/dspace-api/src/main/java/org/dspace/app/mediafilter/ImageMagickThumbnailFilter.java)

**Comment by alawvt** [comment]
@zoto724 reports a Java heap space issue related to this, which he will investigate later
```
2017-04-17 03:01:10,159 ERROR org.dspace.app.mediafilter.PDFFilter @ Error parsing PDF document Java heap space
```

**Comment by cecross1** [comment]
More blank thumbnails seem to be popping up again. Here they are:
- https://vtechworks.lib.vt.edu/handle/10919/83799
- https://vtechworks.lib.vt.edu/handle/10919/83506

The title pages appear to be normal. I haven't seen any other affected items, but it may be worth keeping an eye on.

**Comment by alawvt** [comment]
Resolved in #661.
